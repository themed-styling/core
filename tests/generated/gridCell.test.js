import 'jest-styled-components'

import { gridCell } from '../../src/index.js'

import { testAllOn } from '../utilities.js'

testAllOn(gridCell, 'gridCell')
