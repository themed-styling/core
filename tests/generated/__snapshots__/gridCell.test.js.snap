// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`gridCell renders 0 1`] = `
Array [
  .c0 {
  grid-cell: 0;
}

<div
    className="c0"
  />,
  .c0 {
  grid-cell: 0;
}

<div
    className="c0"
  />,
  .c0 {
  grid-cell: 0;
}

<div
    className="c0"
  />,
  .c0 {
  grid-cell: 0;
}

<div
    className="c0"
  />,
]
`;

exports[`gridCell renders function 1`] = `
Array [
  .c0 {
  grid-cell: hasnot;
}

<div
    className="c0"
  />,
  .c0 {
  grid-cell: has;
}

<div
    className="c0"
  />,
  .c0 {
  grid-cell: hasnot;
}

<div
    className="c0"
  />,
  .c0 {
  grid-cell: has;
}

<div
    className="c0"
  />,
]
`;

exports[`gridCell renders invalid theme path 1`] = `
Array [
  .c0 {
  grid-cell: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  grid-cell: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  grid-cell: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  grid-cell: invalid.path.1;
}

<div
    className="c0"
  />,
]
`;

exports[`gridCell renders negative number 1`] = `
Array [
  .c0 {
  grid-cell: -54881;
}

<div
    className="c0"
  />,
  .c0 {
  grid-cell: -54881;
}

<div
    className="c0"
  />,
  .c0 {
  grid-cell: -42365;
}

<div
    className="c0"
  />,
  .c0 {
  grid-cell: -42365;
}

<div
    className="c0"
  />,
]
`;

exports[`gridCell renders object [breakpoints] 1`] = `
Array [
  @media screen and (min-width:November) {
  .c0 {
    grid-cell: July;
  }
}

@media screen and (min-width:November) {
  .c0 {
    grid-cell: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    grid-cell: September;
  }
}

@media screen and (min-width:August) {
  .c0 {
    grid-cell: November;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {
  .c0 {
    grid-cell: July;
  }
}

@media screen and (min-width:November) {
  .c0 {
    grid-cell: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    grid-cell: September;
  }
}

@media screen and (min-width:August) {
  .c0 {
    grid-cell: November;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    grid-cell: May;
  }
}

@media screen and (min-width:November) {
  .c0 {
    grid-cell: June;
  }
}

@media screen and (min-width:November) {
  .c0 {
    grid-cell: April;
  }
}

@media screen and (min-width:August) {
  .c0 {
    grid-cell: November;
  }
}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    grid-cell: May;
  }
}

@media screen and (min-width:November) {
  .c0 {
    grid-cell: June;
  }
}

@media screen and (min-width:November) {
  .c0 {
    grid-cell: April;
  }
}

@media screen and (min-width:August) {
  .c0 {
    grid-cell: November;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`gridCell renders object [breakpoints] 2`] = `
Array [
  @media May {
  .c0 {
    grid-cell: July;
  }
}

@media April {
  .c0 {
    grid-cell: August;
  }
}

@media January {
  .c0 {
    grid-cell: September;
  }
}

@media April {
  .c0 {
    grid-cell: November;
  }
}

@media June {
  .c0 {
    grid-cell: August;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  @media May {
  .c0 {
    grid-cell: July;
  }
}

@media April {
  .c0 {
    grid-cell: August;
  }
}

@media January {
  .c0 {
    grid-cell: September;
  }
}

@media April {
  .c0 {
    grid-cell: November;
  }
}

@media June {
  .c0 {
    grid-cell: August;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  @media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    grid-cell: January;
  }
}

@media April {
  .c0 {
    grid-cell: December;
  }
}

@media January {
  .c0 {
    grid-cell: April;
  }
}

@media April {
  .c0 {
    grid-cell: May;
  }
}

@media June {
  .c0 {
    grid-cell: June;
  }
}

<div
    className="c0"
  />,
  @media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    grid-cell: January;
  }
}

@media April {
  .c0 {
    grid-cell: December;
  }
}

@media January {
  .c0 {
    grid-cell: April;
  }
}

@media April {
  .c0 {
    grid-cell: May;
  }
}

@media June {
  .c0 {
    grid-cell: June;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`gridCell renders postitive number 1`] = `
Array [
  .c0 {
  grid-cell: 54881;
}

<div
    className="c0"
  />,
  .c0 {
  grid-cell: 54881;
}

<div
    className="c0"
  />,
  .c0 {
  grid-cell: 42365;
}

<div
    className="c0"
  />,
  .c0 {
  grid-cell: 42365;
}

<div
    className="c0"
  />,
]
`;

exports[`gridCell renders string 1`] = `
Array [
  .c0 {
  grid-cell: July;
}

<div
    className="c0"
  />,
  .c0 {
  grid-cell: July;
}

<div
    className="c0"
  />,
  .c0 {
  grid-cell: June;
}

<div
    className="c0"
  />,
  .c0 {
  grid-cell: June;
}

<div
    className="c0"
  />,
]
`;

exports[`gridCell renders valid theme path 1`] = `
Array [
  .c0 {
  grid-cell: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  grid-cell: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  grid-cell: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  grid-cell: DarkSlateBlue;
}

<div
    className="c0"
  />,
]
`;
